pkg/operator/ceph/csi/spec.go

```diff
@@ -96,7 +96,7 @@ var (
 // manually challenging.
 var (
 	// image names
-	DefaultCSIPluginImage   = "quay.io/cephcsi/cephcsi:v3.0.0"
+	DefaultCSIPluginImage   = "quay.io/cephcsi/cephcsi:v3.1.0"
```

Comment:
need to add same to https://github.com/rook/rook/blob/master/pkg/operator/ceph/csi/version.go#L32